frontend/app/controllers/Joiner.scala

```diff
@@ -132,6 +132,10 @@ object Joiner extends Controller with ActivityTracking
     (for {
       identityUser <- identityService.getIdentityUserView(request.user, identityRequest)
     } yield {
+      tier match {
+        case t: Tier.Supporter => MembersDataAPI.Service.addBehaviour(request, "enterPaidDetails.show")
+        case _ =>
```

Comment:
Is this a good Scala style? It seems weird to have an empty right side in a case of a pattern match.